docs/07_tools/02_eosio-helm/index.md

```diff
@@ -0,0 +1,87 @@
+---
+content_title: EOSIO Nodeos Helm Charts
+link_text: EOSIO Nodeos Helm Charts
+---
+
+## Summary
+
+* [Overview](#overview)
+* [Installation](#installation)
+    * [Local Deployment](#local-deployment)
+    * [Cloud Deployment](#cloud-deployment)
+* [Configuration](#configuration)
+    * [Customizing Values](#customizing-values)
+    * [Configuring Cloud Provider](#configuring-cloud-provider)
+* [Tutorial](#tutorial)
+
+## Overview
+
+EOSIO Helm Charts provide direct access to infrastructure-as-code (IaC) for rapid deployment of EOSIO nodes via Kubernetes. It includes support for common cloud providers such as Amazon Web Services (AWS) and Google Cloud Platform (GCP). EOSIO Helm Charts currently includes three charts:
+
+  * `eosio`: provides the configuration umbrella for the EOSIO ecosystem.
+  * `eosio-common`: provides scaffolding for infrastructure extensions support.
+  * `eosio-nodeos`: provides the configuration options for the EOSIO node(s).
+
+[[info | Note]]
+| The [EOSIO Web IDE](https://github.com/EOSIO/eosio-web-ide) is another tool that allows developers to get an EOSIO network up and running quickly. It also allows developers to build and deploy EOSIO applications smoothly from within a Docker-enabled web-based IDE.
+
+## Installation
+
+EOSIO Helm Charts can be installed and run locally with a Docker Desktop backend, or run remotely with a cloud deployment. For quick prototyping and rapid testing you may want a local deployment. For production environments or distributed testing, a cloud deployment is recommended.
```

Comment:
> For quick prototyping and rapid testing you may want a local deployment. For production environments or distributed testing, a cloud deployment is recommended.

is "you may want" and "is recommended" passive? anyhow I would love  a more direct and simple approach, e.g. `we recommend this and that`.